library/templates/address_display.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/**
+ * Handles the display of the address list datatype in LBF
+ * Updated to use ContactService and ContactAddressService
+ *
+ * @package   OpenEMR
+ * @link      https://www.open-emr.org
+ *
+ * @author    David Eschelbacher <psoas@tampabay.rr.com>
+ * @author    Stephen Nielson <snielson@discoverandchange.com>
+ * @copyright Copyright (c) 2022 David Eschelbacher <psoas@tampabay.rr.com>
+ * @copyright Copyright (c) 2022 Discover and Change, Inc. <snielson@discoverandchange.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
```

Comment:
odd to have a script that runs without globals
(if case, would at least ensure it matches some sort of variable from the calling script (assuming this is a template); would also consider this for any other scripts in this PR that do the same thing)

Reply:
yes, would check for presence of passed variable from calling script to ensure secure.